pkgs/by-name/gh/ghunt/package.nix

```
@@ -38,6 +38,8 @@ python3.pkgs.buildPythonApplication rec {
     python-dateutil
     rich
     trio
+    packaging
+    h2
   ];
```

`h2` is an extra dep of [`httpx`](https://github.com/mxrch/GHunt/blob/master/requirements.txt#L2C7-L2C12).

```suggestion
    packaging
  ] ++ httpx.optional-dependencies.http2;
```